Documentation/Reference/ScreenshotContainer/Index.rst

```diff
@@ -0,0 +1,126 @@
+:navigation-title: Screenshot container
+
+..  include:: /Includes.rst.txt
+..  _screenshot-container:
+
+==================================
+TYPO3 Screenshots Docker container
+==================================
+
+We have prepared a Docker container that you can use to create screenshots
+for the official documentation, which already follows out
+`Guidelines for screenshots <https://docs.typo3.org/permalink/h2document:automatic-screenshots>`_.
+
+..  contents:: Table of content
+
+..  seealso::
+
+    *   `Guidelines for screenshots in the official documentation <https://docs.typo3.org/permalink/h2document:automatic-screenshots>`_
+    *   `linawolf/typo3-screenshots <https://hub.docker.com/repository/docker/linawolf/typo3-screenshots/general>`_
+    *   `Ready to use Project for Screenshots (GitHub) <https://github.com/TYPO3-Documentation/site-introduction>`_
+
+..  _screenshot-quick:
+
+Quick screenshots using Docker
+==============================
+
+You can run the screenshot project locally using the prebuilt Docker image:
+
+..  code-block:: bash
+
+    docker run -d --name typo3-screenshots -p 8080:80 linawolf/typo3-screenshots
+
+    # Follow the logs to track progress:
+    docker logs -f typo3-screenshots
+
+Once setup is complete, open:
+
+http://localhost:8080/typo3
+
+..  note::
+
+    The container resets on every run. It is ideal for reproducible test
+    environments and fresh screenshots.
+
+..  _screenshot-reset:
+
+Resetting the container
+=======================
+
+To start fresh, you can stop and remove the running container, then launch it
+again.
+
+..  code-block:: bash
+
+    docker stop typo3-screenshots
+    docker rm typo3-screenshots
+    docker run -d --name typo3-screenshots -p 8080:80 linawolf/typo3-screenshots
+
+This gives you a clean TYPO3 instance every time.
+
+..  _screenshot-ssh:
+
+Accessing the container shell
+=============================
+
+You can access the container's shell to explore the file system, install
+additional extensions, or run TYPO3 CLI commands:
+
+..  code-block:: bash
+
+    docker exec -it typo3-screenshots bash
```

Comment:
Within your container you only work with "/bin/sh". Are you sure "bin/bash" is available?

Reply:
At least it worked like this for me